ci/goreleaser/goreleaser.yml

```diff
@@ -79,56 +60,43 @@ dockers:
     - "coprocess"
     - "tyk.conf.example"
 
-# Build tykio/xDH_REPO, cloudsmith/xCOMPATIBILITY_NAME (arm64)
+# Build gateway hybrid container amd64
 - ids:
     - std
   image_templates:
-    - "tykio/tyk-gateway:{{ .Tag }}-arm64"
-    - "docker.tyk.io/tyk-gateway/tyk-gateway:{{ .Tag }}-arm64"
+    - "tykio/tyk-hybrid-docker:{{ .Tag }}-amd64"
   build_flag_templates:
-    - "--build-arg=PORTS=8080"
-    - "--platform=linux/arm64"
+    - "--platform=linux/amd64"
     - "--label=org.opencontainers.image.created={{.Date}}"
-    - "--label=org.opencontainers.image.title={{.ProjectName}}-arm64"
+    - "--label=org.opencontainers.image.title={{.ProjectName}}-hybrid"
     - "--label=org.opencontainers.image.revision={{.FullCommit}}"
     - "--label=org.opencontainers.image.version={{.Version}}"
   use: buildx
-  goarch: arm64
+  goarch: amd64
   goos: linux
-  dockerfile: ci/Dockerfile.std
+  dockerfile: ci/images/hybrid/Dockerfile
```

Comment:
I've just noticed the base image here is still buster, recent CVE resolutions have only been aimed at bumping the Dockerfile.std/slim environments. When we discussed which images we release, the hybrid docker one was not caught. Is there some kind of test suite that verifies the fidelity for the tyk-hybrid-docker images, so we may bump the base image for it?

Reply:
Yes, we did discuss this, and whether we should bump it to bullseye. There is no test suite that  specifically verifies the hybrid images AFAIK.   Apparently, the legacy cloud is the only place where it's being used. So at least for now, it was decided to keep this as it is.